src/Components/Publishing/Partner/PartnerBlock.tsx

```diff
@@ -1,9 +1,9 @@
 import { unica } from "Assets/Fonts"
+import { pMedia } from "Components/Helpers"
 import React from "react"
 import track from "react-tracking"
```

Comment:
If the tracks in this module adhere to the unified schema we’ve been trying to define, then you need to make sure to import our typed version of react-tracking from.

CC @eessex

Reply:
I don't think this is blocking for this PR -- we'd need to add the values for these actions into the schema, maybe best to ticket updates to analytics for another date since @ansor4 is no longer on Grow.

Reply:
Also heads up @alloy added a follow up ticket to update analytics https://artsyproduct.atlassian.net/browse/GROW-874

Reply:
Thanks!